frontend/app/controllers/Joiner.scala

```diff
@@ -120,7 +120,7 @@ class Joiner(
       case Some(user) => for {
         fullUser <- identityService.getFullUserDetails(user.minimalUser)(IdentityRequest(request))
         primaryEmailAddress = fullUser.primaryEmailAddress
-        displayName = fullUser.publicFields.displayName
+        displayName = DisplayName(fullUser)
```

Comment:
Why do we have `DisplayName.apply` and `DisplayName.fallbackFullname`?

Reply:
Speaking to @dominickendrick and Rosie the plan was to display the Identity username to users, falling back to first name in place of display name (where this is preferable on rendered web pages).

However, for `updateSalesforceContactWithMembership` it seems strange just passing a first name to Salesforce. Sending a first and last name seem more appropriate, maybe we should go with this instead and never pass a username to Salesforce 🤔 

The closest logic to the old functionality would be username, falling back to "$firstName $secondName"

Reply:
This is all effectively dead code since we switched the acquisition flows off so I wouldn't spend too long worrying about the perfect solution

Reply:
@rupertbates  I'll leave as is then. Is there a plan for this code to be removed or the project to be decomissioned?